### PR TITLE
Updated dependency to symfony/property-access 2.3

### PR DESCRIPTION
**Pull request by jeremyFreeAgent**
This PR make the extension compatible with `symfony/property-access` version `2.3`.

### CODE REVIEW

**Comment by tyx** [comment]
But we lose `the JSON node  should not exist` assertion ?!

File: composer.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
         "behat/behat": "~2.5.0",
         "atoum/atoum": "~1.0",
         "guzzle/http": "~3.9",
-        "symfony/property-access": "~2.4",
+        "symfony/property-access": "~2.3.0",
```
Comment (stephpy) [inline composer.json]: Why ?

**Comment by jeremyFreeAgent** [comment]
Yes... But there is another issue. Can get the value of a node with a `-`. So perhaps we must not continue that lead. I have found a way to have `symfony/property-access` version `2.4` on my project. We must move forward!

**Comment by tyx** [comment]
\o/

**Pull request closed without merging**